Sources/Controllers/MessagesViewController.swift

```diff
@@ -56,6 +56,22 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
     /// The default value of this property is `false`.
     open var maintainPositionOnKeyboardFrameChanged: Bool = false
 
+    /// Display the date of message by swiping left.
+    /// The default value of this property is `false`.
+    open var displayTimeBySwipingLeft: Bool = false {
```

Comment:
nit: I think I'd prefer more descriptive name for this property? Something like `showMessageTimestampOn[Pan/Swipe]Left` 🤷